modules/core/src/main/scala/org/tessellation/modules/Daemons.scala

```diff
@@ -25,15 +25,26 @@ object Daemons {
     healthChecks: HealthChecks[F],
     nodeId: PeerId,
     cfg: AppConfig
-  ): F[Unit] =
+  ): F[Unit] = {
+    val c = cfg.download.peerDiscoveryDelay
```

Comment:
can you use a better name for this? it's not within a simple function scope, so it would be better to name it

Reply:
Renamed to `pddConfig`.